docs/src/extend/firmware-management/building-image/rugpi/tutorial.md

```diff
@@ -48,92 +70,103 @@ Feel free to clone the project if you want to make your own customizations, howe
     :::
 
-3. Commit the changes
+4. Optional: Add Wifi ssid/password - only required for the initial flashing of the device
+
+    If your device does not have an ethernet adapter, or you the device to connect to a Wifi network for onboarding, then you will have to add the Wifi credentials to the `.env` file.
 
-4. You can now add any custom recipes to your project if you want to include additional packages in your image, or configure items.
+    Below shows the environment variables that should be added to the `.env` file.
+
+    ```sh
+    SECRETS_WIFI_SSID=example
+    SECRETS_WIFI_PASSWORD=yoursecurepassword
+    ```
+
+    **Note**
+
+    The Wifi credentials only need to be included in the image that is flashed to the SD card. Subsequent images don't need to included the Wifi credentials, as the network connection configuration files are persisted across images.
+
+    If an image has Wifi credentials baked in, then you should not make this image public, as it would expose your credentials! 
+
+    
 
-    See the [Rugpi System Customization](https://oss.silitics.com/rugpi/docs/guide/system-customization) docs for details on how to do this or follow clues from some of the existing recipes included in the [tedge-rugpi-images](https://github.com/thin-edge/tedge-rugpi-image) project.
 
 ### Building an image
 
-:::info
-Currently building is only supported on a Linux environment. It is strongly encouraged to use the Github workflow to build the image.
-:::
+Building an image will produce a `.xz` file which can either by flashed to an SD card (or device). Afterwards your device will be enable to apply the same type of image via OTA updates.
+
+Images can be built in MacOS or Linux environments (including WSL 2), however if you have problems building the image, then you can build the images using the Github workflow after forking the project.
+
 
 1. Build an image for your device (machine)
 
     ```sh tab={"label":"Pi\tZero"}
-    just IMAGE_ARCH=armhf PROFILE=armhf VARIANT=pi01 build-all
+    just build-pizero
     ```
 
     ```sh tab={"label":"Pi\t1"}
-    just IMAGE_ARCH=armhf PROFILE=armhf VARIANT=pi01 build-all
+    just build-pi1
     ```    
 
     ```sh tab={"label":"Pi\t2"}
-    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi023 build-all
+    just build-pi2
     ```
     ```sh tab={"label":"Pi\t3"}
-    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi023 build-all
+    just build-pi3
     ```
     ```sh tab={"label":"Pi\tZero2W"}
-    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi023 build-all
+    just build-pizero2w
     ```
 
     ```sh tab={"label":"Pi\t4\t(With\tFirmware)"}
-    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi4 build-all
+    just build-pi4-include-firmware
     ```
 
     ```sh tab={"label":"Pi\t4\t(Without\tFirmware)"}
-    just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi45 build-all
+    just build-pi4
```

Comment:
How can I build an image with Wifi credentials?

Before this change, I was setting an explicit `PROFILE=wifi`.

Reply:
The instructions for wifi should be covered in the previous step where the `.env` file is used...now you don't have to explicit activate a "wifi" profile, you just have to provide the ssid/password via the `.env` file...which makes building different images much easier and you don't have to change any commit files.

Reply:
Perfect